### DISCUSSION TITLE
Add permission node for itemname with color code restriction

### DISCUSSION THREAD

**Discussion by johnsoncui02** [Zrips/CMI#3718] [Feature Requests]
basically add a permission node for cmi.command.itemname

but with something like cmi.command.itemname.nocolor

to restrict players from using color codes when renamining items. They would all essentially be defaulted to white colors.

**Comment by CloudeLecaw** [comment]
Without the anvil color permissions they are not allowed to use colors on anvil

> 
> cmi.anvil.colors – Allows to use colors when renaming item
> cmi.title.colors – Allows to use colors in titlemsg command
> cmi.tag.color – Tagged player name gets colorized
> cmi.colors.[type].* – Allows all color usage in particular areas
> cmi.colors.[type].[colorname] – Allows color usage in particular areas. Types: publicmessage, privatemessage, nickname, signs, books, me

**Comment by johnsoncui02** [comment]
I'm referring to the command /itemname specifically.
